bagel/tests/test_cli_pheno.py

```diff
@@ -255,6 +255,41 @@ def test_unused_missing_values_raises_warning(
         assert warn_substring in str(w[0].message.args[0])
 
 
+@pytest.mark.parametrize(
+    "pheno_file,dictionary_file",
+    [
+        ("example2.csv", "example2.json"),
+        ("example16.tsv", "example16.json"),
+    ],
+)
+def test_providing_csv_file_raises_error(
+    pheno_file,
+    dictionary_file,
+    runner,
+    test_data,
+    tmp_path,
+):
+    """Providing a .csv file or a file with .tsv extension but incorrect encoding should be handled with an infomrative error."""
```

Comment:
typo missed by codespell:

```suggestion
    """Providing a .csv file or a file with .tsv extension but incorrect encoding should be handled with an informative error."""
```